src/OnlineSales/Migrations/20230525204710_ResetDoaminCheckState.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace OnlineSales.Migrations
+{
+    /// <inheritdoc />
+    public partial class ResetDoaminCheckState : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.Sql("UPDATE  domain SET dns_check = null, http_check = null, mx_check = null");
```

Comment:
@Exmirai please remove this migration as it is very dangerous to reset these variables to null because we can potentially lose lots of valuable information. Please do not even reset http_check. It is better to manually re-validate these domains where we see errors (separate code will be implemented later to do it).